src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -246,6 +246,8 @@ class WiFiManager
     uint32_t mConnectionRecoveryTimeMs{ kConnectionRecoveryMinIntervalMs };
     bool mRecoveryArmed{ false };
     bool mApplicationDisconnectRequested{ false };
+    // TODO Workaround for recovery mechanism to wait before the next scan request until the WiFi supplicant is not busy.
+    System::Clock::Seconds32 WifiSupplicantWorkaroundTime {8};
```

Comment:
Please keep the notation scheme, add make it const or constexpr. Looks like the formatting is also a bit off
```suggestion
    static constexpr System::Clock::Seconds32 kWifiSupplicantWorkaroundTime{ 8 };
```